spring-cloud-starter-stream-processor-pose-estimation/src/test/java/org/springframework/cloud/stream/app/pose/estimation/processor/PoseEstimationTensorflowProcessorIntegrationTests.java

```diff
@@ -0,0 +1,132 @@
+						"{\"limbs\":[{\"score\":6.4001913,\"from\":{\"type\":\"nose\",\"y\":24,\"x\":416},\"to\":{\"type\":\"rEye\",\"y\":16,\"x\":408}},{\"score\":8.033572,\"from\":{\"type\":\"neck\",\"y\":48,\"x\":432},\"to\":{\"type\":\"rHip\",\"y\":120,\"x\":424}},{\"score\":8.621042,\"from\":{\"type\":\"neck\",\"y\":48,\"x\":432},\"to\":{\"type\":\"lHip\",\"y\":120,\"x\":432}},{\"score\":5.836733,\"from\":{\"type\":\"rShoulder\",\"y\":48,\"x\":424},\"to\":{\"type\":\"rElbow\",\"y\":88,\"x\":424}},{\"score\":9.344168,\"from\":{\"type\":\"rKnee\",\"y\":176,\"x\":432},\"to\":{\"type\":\"rAnkle\",\"y\":232,\"x\":432}},{\"score\":8.949261,\"from\":{\"type\":\"rHip\",\"y\":120,\"x\":424},\"to\":{\"type\":\"rKnee\",\"y\":176,\"x\":432}},{\"score\":8.429034,\"from\":{\"type\":\"neck\",\"y\":48,\"x\":432},\"to\":{\"type\":\"nose\",\"y\":24,\"x\":416}},{\"score\":8.87405,\"from\":{\"type\":\"lHip\",\"y\":120,\"x\":432},\"to\":{\"type\":\"lKnee\",\"y\":176,\"x\":440}},{\"score\":6.4418483,\"from\":{\"type\":\"neck\",\"y\":48,\"x\":432},\"to\":{\"type\":\"rShoulder\",\"y\":48,\"x\":424}},{\"score\":9.649043,\"from\":{\"type\":\"lKnee\",\"y\":176,\"x\":440},\"to\":{\"type\":\"lAnkle\",\"y\":224,\"x\":472}}]}," +
+						"{\"limbs\":[{\"score\":8.600379,\"from\":{\"type\":\"neck\",\"y\":48,\"x\":328},\"to\":{\"type\":\"lShoulder\",\"y\":48,\"x\":320}},{\"score\":5.776266,\"from\":{\"type\":\"rShoulder\",\"y\":56,\"x\":344},\"to\":{\"type\":\"rEar\",\"y\":32,\"x\":360}},{\"score\":6.8949876,\"from\":{\"type\":\"neck\",\"y\":48,\"x\":328},\"to\":{\"type\":\"lHip\",\"y\":128,\"x\":304}},{\"score\":9.813445,\"from\":{\"type\":\"rKnee\",\"y\":192,\"x\":336},\"to\":{\"type\":\"rAnkle\",\"y\":248,\"x\":336}},{\"score\":7.878962,\"from\":{\"type\":\"rHip\",\"y\":128,\"x\":328},\"to\":{\"type\":\"rKnee\",\"y\":192,\"x\":336}},{\"score\":7.7970695,\"from\":{\"type\":\"rShoulder\",\"y\":56,\"x\":344},\"to\":{\"type\":\"rElbow\",\"y\":96,\"x\":360}},{\"score\":9.607412,\"from\":{\"type\":\"lHip\",\"y\":128,\"x\":304},\"to\":{\"type\":\"lKnee\",\"y\":184,\"x\":304}},{\"score\":7.85779,\"from\":{\"type\":\"neck\",\"y\":48,\"x\":328},\"to\":{\"type\":\"rHip\",\"y\":128,\"x\":328}},{\"score\":9.062571,\"from\":{\"type\":\"lKnee\",\"y\":184,\"x\":304},\"to\":{\"type\":\"lAnkle\",\"y\":240,\"x\":296}},{\"score\":7.347301,\"from\":{\"type\":\"rElbow\",\"y\":96,\"x\":360},\"to\":{\"type\":\"rWist\",\"y\":104,\"x\":400}},{\"score\":8.256059,\"from\":{\"type\":\"neck\",\"y\":48,\"x\":328},\"to\":{\"type\":\"rShoulder\",\"y\":56,\"x\":344}}]}]\n");
+
+				JSONAssert.assertEquals(expected, new JSONArray(received.getPayload()), false);
+			}
+		}
+
+		@TestPropertySource(properties = {
+				"tensorflow.mode=header",
+				//"tensorflow.model=file:/Users/ctzolov/Dev/projects/tf-pose-estimation/models/graph/mobilenet_thin/graph_opt.pb",
+				"tensorflow.model=file:/Users/ctzolov/Dev/projects/tf-pose-estimation/models/graph/cmu/graph_opt.pb",
```

Comment:
@tzolov Can we replace this with some local file reference?

Reply:
Good point. 
I will fix this for the other processors tests as well